libs/protocol-client/src/lib/client/contribution.ts

```diff
@@ -16,6 +16,9 @@ import {
 import { GraphQLClient } from 'graphql-request';
 import { paginate } from '../utils';
 
+const REQUIRED_CONFIRMATIONS = 1;
+const ERROR_CHAIN_ID = 'Failed to fetch on chain Id';
```

Comment:
It may be better to abstract this into a new error type. Something like `class ChainIdError extends Error {
  constructor(message: string) {
    super(message || 'Failed to fetch on chain Id')
}
}`